lib/src/dap/dap_resolver.dart

```diff
@@ -1,6 +1,6 @@
 import 'dart:convert';
 
-import 'package:dap/src/dap.dart';
+import 'package:dap/src/dap/dap.dart';
```

Comment:
damn `package:dap/src/dap/dap.dart`? do we need a `dap` dir inside `src` as well?

Reply:
hmm prob not, was following how we consolidated exported files in `web5-dart` and `tbdex-dart` (e.g. `package:web5/src/dids/did.dart`) but def isn't necessary in this case